src/components/Layout/LayoutWrapper.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import Header from './Header';
+import HomeView from './HomeView';
+import { SearchWrapper } from '../../wrappers/SearchWrapper';
+
+const StyledLayoutWrapper = styled.div`
+  background: ${({ theme }) => theme.color.background};
+  width: 100%;
+  height: 100%;
+`;
+
+const LayoutWrapper = ({ children }) => {
+  const { theme } = useContext(ThemeContext);
+
+  return (
+    <StyledLayoutWrapper data-testid="layout-wrapper" theme={theme} role="application">
```

Comment:
It's not necessary to get the current theme from the context so your styled-components can use it. Since you're wrapping your App using the `ThemeProvider` all your `styled-components` will automatically receive the current theme in their props. https://styled-components.com/docs/advanced

```suggestion
const StyledLayoutWrapper = styled.div`
  background: ${({ theme }) => theme.color.background};
  width: 100%;
  height: 100%;
`;

const LayoutWrapper = ({ children }) => {
  return (
    <StyledLayoutWrapper data-testid="layout-wrapper" role="application">
```